var/spack/repos/builtin/packages/xios/package.py

```diff
@@ -45,6 +46,13 @@ class Xios(Package):
 
     @when('%clang')
     def patch(self):
+        self.patch_llvm()
+
+    @when('%apple-clang')
+    def patch(self):
+        self.patch_llvm()
+
+    def patch_llvm(self):
```

Comment:
Same comment on hack here

Reply:
I would leave this like that, maybe fix it later.